Library/OcConsoleLib/TextOutputBuiltin.c

```diff
@@ -424,6 +425,12 @@ AsciiTextReset (
 
   if (EFI_ERROR (Status)) {
     gBS->RestoreTPL (OldTpl);
+    if (Verbose) {
+      DEBUG ((DEBUG_INFO, "OCC: ASCII Text Reset HandleProtocolFallback - %r\n", Status));
+
+      return Status
```

Comment:
Why return Status here?

Reply:
Related to Task 8 that I added.

As it stands, this should log things in OpenCore clearly but need a way for the 'raw' statuses to get to a 3rd party calling OcUseBuiltinTextOutput. I also need to add the same to the other verbose if block to achieve this.

OpenCore does not handle the status from OcUseBuiltinTextOutput so should not be an issue on that front.

OK with you?

Reply:
Changes approach to leave internal OpenCorePkg calls unchanged. External calls to can be made to new OcUseBuiltinTextOutputEx Function if raw status is required.

Will build and test.